Sources/KituraNet/ClientResponse.swift

```diff
@@ -37,19 +37,19 @@ public class ClientResponse {
     /// Minor version of HTTP of the response
     public var httpVersionMinor: UInt16?
 
-    internal var _headers: HTTPHeaders?
+    internal var httpHeaders: HTTPHeaders?
```

Comment:
Did you mean to rename this?

Reply:
@ianpartridge Yes, by rules it can contain only alphanumeric characters.